src/ProjectTemplates/Shared/AspNetProcess.cs

```diff
@@ -171,12 +171,19 @@ public async Task ContainsLinks(Page page)
             else
             {
                 Assert.True(string.Equals(anchor.Href, expectedLink), $"Expected next link to be {expectedLink} but it was {anchor.Href}.");
-                var result = await RetryHelper.RetryRequest(async () =>
+
+                if (!string.Equals(anchor.Protocol, "https:"))
```

Comment:
For completeness here, can we check also that the host is not `localhost` or `127.0.0.1`? Just in case we emit any absolute link in the templates.

Reply:
Makes sense - I've added a check for `localhost`.

I haven't added one for `127.0.0.1` as it would be extraordinary and certainly wrong for someone to add such a link to the templates. The tests explicitly list all the URLs we expect to see, so we'd definitely know about it if someone added a link like that.

Reply:
@SteveSandersonMS that's fair. I was thinking more in terms of absolute links generated automatically, not something written up by hand.